sdk/collections/lookup-map.js

```diff
@@ -0,0 +1,33 @@
+import * as near from '../api'
+
+export class LookupMap {
+    constructor(keyPrefix) {
+        this.keyPrefix = keyPrefix
+    }
+
+    containsKey(key) {
+        let storageKey = this.keyPrefix + key
+        return near.jsvmStorageHasKey(storageKey)
+    }
+
+    get(key) {
+        let storageKey = this.keyPrefix + key
+        return near.jsvmStorageRead(storageKey)
```

Comment:
Yes. The other way is to have a get and getRaw like rust. This is actually `getRaw`. But I do want to decouple deserialization with the actual data structure as much as possible, so that 1) user can choose their own serialization; 2) the implementation of each data structure is simply the data structure part.

I think a proper way to do this need more discuss and consideration on trade offs. One possible solution is to let user inherit LookupMap to set their serialization. In this approach we may also need to provide some default impl, a JSONLookupMap, for example. The other approach is to make serialize as mixin, this is closest to Rust's traits.

But no matter which more well-designed approach we'll use, the greatest common divisor version can't be wrong. So I decide on this version for our first release.